SusyCommon/TriggerMap.h

```diff
@@ -0,0 +1,118 @@
+#ifndef SusyCommon_TriggerMap_h
+#define SusyCommon_TriggerMap_h
+
+#include <iostream>
+#include <string>
+#include <map>
+
+
+#include "TBits.h"
+
+using namespace std;
```

Comment:
If possible, avoid using `using namespace` in header
http://stackoverflow.com/questions/5849457/using-namespace-in-c-headers